src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
```

Comment:
The `adder` function should not accept any arguments other than numbers. Although the task instructions mention that there's no need to check if the argument is a number, it's usually a good practice to ensure that the function handles only the expected data types to prevent potential bugs in the future. However, as per the task instructions, we can skip this check.